CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
 # NEXT RELEASE
 
 ### Enhancements
-* <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))
-* None.
+* Add support for chunked transfer encoding when using `HTTPParser`.
```

Comment:
I think this is more of an internal change - no SDKs (other than C++) should be outwardly affected by this.